Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM registry.suse.com/bci/bci-base:15.3
+RUN zypper -n rm container-suseconnect && \
+    zypper install -y iptables=1.8.7 && \
+    zypper -n clean -a && rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/*
+COPY bin/harvester-network-controller /usr/bin/
+CMD ["harvester-network-controller"]
```

Comment:
suggest to follow harvester, use `tini` as process 0 in container

https://github.com/krallin/tini

https://github.com/harvester/harvester/blob/master/package/Dockerfile

https://github.com/harvester/harvester/blob/master/package/entrypoint.sh

consider to optimize in another issue.

Reply:
Sorry, it should not be here. I just copy it under the root directory for debugging.